### PR TITLE
Fix edge case when filesPayload message is nil

### PR DESCRIPTION
**Pull request by crimsonred**
fix(Fetch): fix paged history

Fix edge case in Fetch response when filesPayload message is nil.

### CODE REVIEW

File: files_common.go [hunk 1]
```diff
@@ -65,18 +65,22 @@ func ParseFileInfo(filesPayload map[string]interface{}) (PNFileDetails, PNPublis
 
 	//"message":{"text":"test file"},"file":{"name":"test_file_upload_name_32899","id":"9076246e-5036-42af-b3a3-767b514c93c8"}}
 	if o, ok := filesPayload["file"]; ok {
-		data = o.(map[string]interface{})
-		if d, ok := data["id"]; ok {
-			resp.PNFile.ID = d.(string)
-		}
-		if d, ok := data["name"]; ok {
-			resp.PNFile.Name = d.(string)
+		if o != nil {
```
Comment (kleewho) [inline files_common.go:68]: Isn't `file` part mandatory and if we don't have it's actually a problem someplace else? If that's true I would rather let it fail here

Comment (crimsonred) [reply files_common.go:68]: It's not mandatory in Fetch.  There can be message that are not associated with Files.

**Review by kleewho — APPROVED** [review]

File: files_common.go (hunk 1 above)
Comment (kleewho) [reply files_common.go:68]: Right

**Comment by crimsonred** [comment]
@client-engineering-bot help

**Comment by crimsonred** [comment]
@client-engineering-bot release

**Review by kleewho — APPROVED** [review]

**Pull request merged**